internal/engine/configs/configs_controller_test.go

```diff
@@ -219,6 +233,29 @@ func (f *ccFixture) setManifestResult(m model.Manifest) {
 	}
 }
 
+func (f *ccFixture) onChange() {
+	_ = f.cc.OnChange(f.ctx, f.st, store.LegacyChangeSummary())
+}
+
+// Wait for the next item on the workqueue, then run reconcile on it.
+func (f *ccFixture) popQueue() {
+	f.T().Helper()
+
+	done := make(chan struct{})
+	go func() {
+		item, _ := f.q.Get()
+		_, _ = f.tfr.Reconcile(f.ctx, item.(reconcile.Request))
```

Comment:
Would it be practical to assert on no errors here?

Reply:
good idea! done